### DISCUSSION TITLE
help setting up s3 storage please

### DISCUSSION THREAD

**Discussion by hobo1618** [pocketbase/pocketbase#679] [Q&A] [answered]
I'm trying to connect to an s3 bucket but getting a `Failed to establish S3 connection` warning when I save the changes. This is what I've entered so far:

![Screenshot from 2022-09-30 20-59-58](https://user-images.githubusercontent.com/24655969/193338777-c99ab283-f9da-4530-af74-ce0767f59f65.png)

I know pretty much nothing about AWS or S3 so any help would be much appreciated.

Thanks!

**Comment by ganigeorgiev** [comment, marked as answer]
The config from the screenshot doesn't look properly formatted.

For "Endpoint" I believe the correct one should be just `s3.eu-central-1.amazonaws.com`.

For "Bucket" - keep only the plain bucket name, eg. `bucket-name`.

For "Region" - keep only the region identifier, eg. `eu-central-1`.

_I'll move the issue to Q&A discussion._

**Reply by hobo1618** [reply]
brilliant, thanks so much
